sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2/async_samples/sample_analyze_business_cards_async.py

```diff
@@ -165,19 +165,17 @@ async def main():
     try:
         asyncio.run(main())
     except HttpResponseError as error:
+        print("For more information about troubleshooting errors, see the following guide: "
+              "https://aka.ms/azsdk/python/formrecognizer/troubleshooting")
         # Examples of how to check an HttpResponseError
         # Check by error code:
         if error.error is not None:
-            if error.error.code == "InvalidRequest":
-                print(f"Received an invalid request error: {error.error}")
-                sys.exit(1)
```

Comment:
Isn't this a pretty common error? Should we keep it?

Reply:
The way I decided to still showcase invalid request information was by checking the message for invalid request on line 177, kind of just showing options for how to access the error. If you think it's better to have invalid request here and some other type in the message check I can switch it up. Let me know!

Reply:
Okay, I just don't have a feel for the most common error codes returned. Seems we'd want to demonstrate capturing those if possible so we don't end up in the "else" case

Reply:
I'll add the check for the invalidrequest error code once again just to make sure we're catching all of those